v2/yarpcjson/outbound.go

```diff
@@ -47,41 +46,42 @@ type Client struct {
 // Returns the response or an error if the request failed.
 func (c Client) Call(ctx context.Context, procedure string, reqBody interface{}, resBodyOut interface{}, opts ...yarpc.CallOption) error {
 	call := yarpc.NewOutboundCall(opts...)
-	treq := yarpc.Request{
+	request := yarpc.Request{
```

Comment:
Nit: Let’s leave a note in CHANGES to audit and align all request/response instances to have the same local variable name throughout the v2 branch. I don’t care whether it’s req/res or request/response or req/resp (but I like resp least).

Reply:
done